hack/minikube-clusters-start.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bashset +x
+
+export KUBECONFIG=${PWD}/state-transfer.kubeconfig
```

Comment:
Do we need this?

Reply:
this was just to make sure that the KUBECONFIG only has two contexts and we don't collide with a user-defined context that is the same name as ours.

Reply:
I think we should drop this for now. We can add notes about overriding KUBECONFIG to control where the file ends up , collisions, etc.

Reply:
dropped